### ISSUE TITLE
Accordion page does not demonstrate JavaScript implementation

### ISSUE THREAD

**Issue by alexhorner** [twbs/bootstrap#37618]
The Accordion page https://getbootstrap.com/docs/5.2/components/accordion/ does not have a Usage via JavaScript section like Modal and Collapse does.

Even though Accordion is documented to use Collapse under the hood, it is not clear how to construct a working Accordion using JavaScript. Using the `Collapse` JavaScript object's constructor on each of the Accordion elements breaks them, and it is not clear how to properly implement the Collapse.

Ideally, a Usage via JavaScript section would be added to Accordion explaining how to either construct an Accordion, or how to correctly construct a `Collapse` object such that all of the Accordion buttons and bodies work correctly, and when opening one Accordion segment, all other Accordion segments close, just like the non-JavaScript-created variant works right now.

This would be very useful for constructing a templated Accordion as a result of an XHR request. Right now, the result of creating a templated Accordion (with unique IDs as appropriate) is a non-functional Accordion.

I have not been able to find a workaround to this issue and I have ended up with a broken, non-functional Accordion with all bodies uncollapsed for now.

**Comment by zvolcsey** [comment]
Hi, can I help add this to the docs?

**Comment by alexhorner** [comment]
> Hi, can I help add this to the docs?

Hi there, obviously I cannot answer this question as it is not my place to say, but if you have a solution, please could you also post it here do I can apply it to my production product?

Thanks!

**Comment by alexhorner** [comment]
Hi there,

Seems the person I responded to previously has deleted their commment,

Are we able to get any update on whether this is even possible please?

Thanks

**Comment by julien-deramond** [comment]
> Are we able to get any update on whether this is even possible please?

Hi @alexhorner. Sorry for the delay but we have tons of things to tackle right now in Bootstrap with a very limited amount of time available. We'll get back to you asap :)

**Comment by Riiiad** [comment]
Can you just copy this and try:

```
import "bootstrap/dist/css/bootstrap.min.css";

// Just leave this include here it seems to be incluing the bootstrap js
import { Dropdown } from 'bootstrap';

document.querySelectorAll('[data-bs-toggle="dropdown"]')
  .forEach(dropdown => {
    new Dropdown(dropdown)
  })

```

**Comment by Riiiad** [comment]
Seems strange, but that will import everything from BS. Just the CSS part is from you, you need to check if it will find this path maybe also try: `import "node_modules/bootstrap/dist/css/bootstrap.min.css";`

**Comment by Riiiad** [comment]
Yes I have the same but it seem at lease for my Vite build everything is included from Bootstrap. Even just the Dropdown includes also Collapse.

If you Accordions are created via JS then just create them before the new Dropdown.

**Comment by alexhorner** [comment]
> Yes I have the same but it seem at lease for my Vite build everything is included from Bootstrap. Even just the Dropdown includes also Collapse.
> 
> If you Accordions are created via JS then just create them before the new Dropdown.

I'm sorry, I don't follow. There are no dropdowns involved here at all.

**Comment by alexhorner** [comment]
Hi,

Just wanted to bump this one, just over a year since the last comment :P

This is still an issue, there does not appear to be any way to activate accordions dynamically using JS in the latest versions,

Thanks

**Comment by mdo** [comment]
Sorry, super late here obviously, but back with v6 work. Closing as accordion is no longer JS-based there.
